tests/test_dse.py

```diff
@@ -1361,17 +1377,14 @@ def test_space_invariant(self):
 
         op = Operator([pde, df])
 
-        # Check code generation
         trees = retrieve_iteration_tree(op)
+
         assert len(trees) == 3
         assert len(trees[0]) == 3  # time, x, y
         assert len(trees[1]) == 2  # time, f
         assert trees[1][0].dim is time
         assert trees[1][1].dim is f
-        assert len(trees[2]) == 1
-        trees = retrieve_iteration_tree(op._func_table['bf0'].root)
-        assert len(trees) == 1
-        assert len(trees[0]) == 4  # f-blocked, f, x, y
+        assert len(trees[2]) == 5
```

Comment:
add comment as in master? what are these 5 Iterations -- `time, f-blocked, f, x, y` ?

Reply:
fixed